docs/cookbook/emails/custom-email-template-per-channel.rst

```diff
@@ -0,0 +1,74 @@
+How to customise email templates per channel?
+=============================================
+
+.. note::
+
+    This cookbook is suitable for a clean :doc:`sylius-standard installation </book/installation/installation>`.
+    For more general tips, while using :doc:`SyliusMailerBundle </components_and_bundles/bundles/SyliusMailerBundle/index>`
+    go to `Sending configurable e-mails in Symfony Blogpost <http://sylius.com/blog/sending-configurable-e-mails-in-symfony>`_.
+
+It is a common use-case to customise email templates depending on a channel in which an action was performed.
+
+1. Pick a template to customize
+
+You can find the list of all email templates and their data on :doc:`emails </book/architecture/emails>` documentation page.
+Then, override that template following our :doc:`template customization </customization/template>` guide.
+
+In this cookbook, let's assume that we want to customize the order confirmation email located at ``@SyliusShopBundle/Email/orderConfirmation.html.twig``.
+This requires creating a new template in ``templates/bundles/SyliusShopBundle/Email/orderConfirmation.html.twig``.
+
+2. Add an ``if`` statement
+
+The simplest customisation might be done with an ``if`` statement in the template.
+
+.. code-block:: php
+
+    <!-- templates/bundles/SyliusShopBundle/Email/orderConfirmation.html.twig -->
+
+    {% block subject %}Topic{% endblock %}
+
+    {% block body %}{% autoescape %}
+        {% if channel.code === 'TOY_STORE' %}
+            Thanks for buying one of our toys!
+        {% else %}
+            Thanks for buying!
+        {% endif %}
+
+        Your order no. {{ order.number }} has been successfully placed.
+    {% endautoescape %}{% endblock %}
+
+3. Extract templates
```

Comment:
Isn't the previous step enough for the first iteration? Can we add a note, that it can be done nicer with dynamic `include`?

Reply:
Added an "optional" label and a bit of description.